### DISCUSSION TITLE
not able to install ruby 2.2.3 on big sur on new Mac pro M1

### DISCUSSION THREAD

**Discussion by BaidyaAnup** [rbenv/ruby-build#1943] [Build failures]
Tried every solution available, still no luck.pls help if any one has done

**Comment by hsbt** [comment]
Unfortunately, Ruby 2.2.x doesn't support with M1 macOS.

**Comment by disbelief** [comment]
Hi @hsbt I'm wondering if the same is true for ruby 1.9.x on M1 macOS? Is every version below 2.2.x unsupported?

**Reply by hsbt** [reply]
The all versions of < Ruby 2.6 don't support M1 macOS.

**Reply by disbelief** [reply]
I see! Bad news for me, but thank you for clarifying so I stop spending time trying to get it working.
